tests/js/server/aql/aql-optimizer-rule-collect-in-cluster.js

```diff
@@ -152,6 +150,26 @@ function optimizerCollectInClusterSuite(isSearchAlias) {
       assertNotEqual(-1, plan.rules.indexOf("collect-in-cluster"));
       assertEqual(["SingletonNode", "EnumerateViewNode", "RemoteNode", "GatherNode", "ScatterNode", "RemoteNode", "EnumerateViewNode", "CollectNode", "RemoteNode", "GatherNode", "CollectNode", "ReturnNode"], nodeTypes);
     },
+    
+    testCountGroups: function () {
+      let query = "FOR doc IN " + c.name() + " COLLECT value = doc.group WITH COUNT INTO length SORT value RETURN { value, length }";
+      let results = AQL_EXECUTE(query);
+      
+      let expected = [];
+      for (let i = 0; i < 10; ++i) {
+        expected.push({ value: "test" + i, length: 100 });
+      }
+      assertEqual(expected.length, results.json.length);
+      assertEqual(expected, results.json);
+
+      let plan = AQL_EXPLAIN(query).plan;
+      let nodeTypes = plan.nodes.map(function (node) {
+        return node.type === 'IndexNode' ? 'EnumerateCollectionNode' : node.type;
+      });
+
+      assertNotEqual(-1, plan.rules.indexOf("collect-in-cluster"));
```

Comment:
We could use `includes()` here instead for better readability:
- [Documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes?retiredLocale=de)

which then would lead to: 
```
assertFalse(plan.rules.includes("collect-in-cluster"));
```

Or to (positive case) :
```
assertTrue(plan.rules.includes("collect-in-cluster"));
```